src/views/Dashboard.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <div class="dashboard">
+      <div class="pipeline saphyr">
+         <router-link :to="{ path: '/saphyr' }" tag="a">
```

Comment:
Can we remove the description too?

Reply:
I'm going to leave it there for now. I would like it to be a placeholder so when we come to UAT it gives people a push in the direction of what they will want in there.